rpc/src/eth_rpc.rs

```diff
@@ -256,6 +256,16 @@ impl EthApiServer for KakarotEthRpc {
         Ok(block_number)
     }
 
+    async fn get_code(&self, _address: Address, _block_number: Option<BlockId>) -> Result<Bytes> {
```

Comment:
What is the format of the Address type? Is it compatible with a Starknet address?